assets/js/main.js

```diff
@@ -137,29 +154,57 @@ $(document).on('click','.state', function() {
 				$('#play-button, #pause-button').toggle();
 			} else {
 				storedSound.pause();
-				currentSong.removeClass('pause-button-img').addClass('play-button-img');	
+				currentSong.removeClass('pause-button-img').addClass('play-button-img');
 				$('#play-button, #pause-button').toggle();
 			} 
 			
 		} else { 
 			if (voy) {
-				$('.artwork-container div').removeClass('pause-button-img').addClass('play-button-img');
-				voy.stop();
-				sounds = [];
+				songReset();
 			}
 
 			SC.stream(tracks[trackIndex]['id'], function(sound){ 
-				$('#play-button').css('display','none');
-				$('#pause-button').css('display','block');
-				currentSong.removeClass('play-button-img').addClass('pause-button-img');
+				buttonToggle();
 				sound.play();
 				sounds[trackIndex] = sound;
 				voy = sound;
-				console.log(voy);
 				showDuration();
 			});	
 		}
-		
+	});
+});
+
+//next song
+$('#next-button').click(function() {
+	songReset();
```

Comment:
this code and the code below is the same minus the trackIndex, DRY up the code and just pass in a trackIndex